health_check.py

```
@@ -0,0 +1,41 @@
+    def __init__(self, bot):
+        self.bot = bot
+
+    @property
+    def name(self) -> str:
+        return "Background Loops"
+
+    @property
+    def critical(self) -> bool:
+        return False  # Individual loops failing is degraded, not critical
+
+    async def run(self) -> HealthCheckResult:
```

Flake8 (7.3.0)

[error] undefined name 'HealthCheckResult'

(F821)